via/get.go

```diff
@@ -22,3 +22,33 @@ func IsConnected(address string) bool {
 
 	return connected
 }
+
+// GetVolume for a VIA device
+func GetVolume(address string) (int, error) {
+
+	defer color.Unset()
+	color.Set(color.FgYellow)
+
+	var command ViaCommand
+	command.Command = "Vol"
+	command.Param1 = "Get"
+
+	log.Printf("Sending command to get VIA Volume to %s", address)
+	// Note: Volume Get command in VIA API doesn't have any error handling so it only returns Vol|Get|XX or nothing
+	// I am still checking for errors just in case something else fails during execution
+	vollevel, err := SendCommand(command, address)
+	//check the error first and then parse
+	if err != nil {
+		return 0, err
+	} else {
+		// parse the returned string for the number that volume is set to
+		// code in common.go so other parts can use the same parser function
+		volint, err := VolumeParse(vollevel)
```

Comment:
Here you can just 
`return VolumeParse(vollevel)`

You don't necessarily need to do the if check below. the default value for an integer is `0` so if `err != nil` then `volint` is going to be 0. if `err == nil` you just want to return the nil error and volint anyway - so they boil down to the same thing.